packages/lzards-api-client/templates/API.hbs

```diff
@@ -0,0 +1,7 @@
+# API
```

Comment:
Pease add more information to this template, similar to https://github.com/nasa/cumulus/blob/master/packages/checksum/templates/README.md.hbs
You can copy some content from
https://github.com/nasa/cumulus/tree/master/packages/lzards-api-client